drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -4499,6 +4499,15 @@ RID RenderingDeviceVulkan::vertex_array_create(uint32_t p_vertex_count, VertexFo
 			if (atf.frequency == VERTEX_FREQUENCY_VERTEX) {
 				// Validate size for regular drawing.
 				uint64_t total_size = uint64_t(atf.stride) * (p_vertex_count - 1) + atf.offset + element_size;
+				if (total_size > buffer->size) {
+					print_line("stride is: " + itos(atf.stride));
```

Comment:
Nitpick but should we use `print_verbose` here instead of `print_line`, or is it unlikely enough that this gets printed?

Reply:
Oof, that should just be removed I think